### PR TITLE
Move TensorRT builder configs to converter build function

### PR DESCRIPTION
**Pull request by pooyadavoodi**

### CODE REVIEW

File: tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc [hunk 1]
```diff
@@ -5614,37 +5634,17 @@ Status ConvertGraphDefToEngine(
   engine->reset();
   if (convert_successfully) *convert_successfully = false;
 
-  // Create the builder.
```
Comment (aaroey) [inline tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc]: Can we move the builder inside BuildCudaEngine as well?

**Review by aaroey — CHANGES_REQUESTED** [review]
Can we move the builder inside BuildCudaEngine as well?

**Comment by pooyadavoodi** [comment]
> Can we move the builder inside BuildCudaEngine as well?

`builder` is needed to create the network in `Converter::Create`.
How about moving it to `Converter::Create`?

**Comment by pooyadavoodi** [comment]
> > Can we move the builder inside BuildCudaEngine as well?
> 
> `builder` is needed to create the network in `Converter::Create`.
> How about moving it to `Converter::Create`?

I moved builder to Converter. Now, the builder is owned by Converter, and it gets created in Converter::Init.

**Review by aaroey — APPROVED** [review]

**Pull request merged**